nikola/nikola.py

```diff
@@ -28,7 +28,6 @@
 
 from __future__ import print_function, unicode_literals
 import io
-from collections import defaultdict
```

Comment:
Looks like it. I'll take a look when I come back

El sáb., jun. 4, 2016 14:55, Chris Warrick <notifications@github.com>
escribió:

> In nikola/nikola.py
> <https://github.com/getnikola/nikola/pull/2364#discussion_r65805487>:
>
> > @@ -28,7 +28,6 @@
> >
> >  from __future__ import print_function, unicode_literals
> >  import io
> > -from collections import defaultdict
>
> Breaks everything!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getnikola/nikola/pull/2364/files/e06dc84de7647df6afee8700b444f73fb0e2384e#r65805487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAGKxY67v-zXJwitNSEQSPBzO4YPB4Cks5qIbwYgaJpZM4IuKiY>
> .
>